osclib/request_splitter.py

```diff
@@ -505,7 +505,7 @@ class StrategySpecial(StrategyNone):
     # Configurable via splitter-special-packages.
     PACKAGES = [
         'gcc',
-        'gcc7',
+        'gcc8',
```

Comment:
This can also be done via `splitter-special-packages` if you want it to go into effect sooner.